spec/models/resource_spec.rb

```diff
@@ -28,7 +28,7 @@
     end
 
     context "existing resource/language combination" do
-      it "adds a new version" do
+      xit "adds a new version" do
```

Comment:
Do you mean for this to remain disabled? If so, perhaps remove the test?

(Same question, retroactively, about spec/services/okta_auth_service_spec.rb:49)